src/tfr_navigation/include/tfr_navigation/navigation_goal_manager.h

```diff
@@ -0,0 +1,34 @@
+#ifndef NAVIGATION_GOAL_MANAGER_H
+#define NAVIGATION_GOAL_MANAGER_H
+#include <ros/console.h>
+#include <ros/ros.h>
+#include <geometry_msgs/Pose.h>
+#include <move_base_msgs/MoveBaseAction.h>
+#include <tfr_msgs/NavigationAction.h>
+#include <cstdint>
+#include <cmath>
+class NavigationGoalManager
+{
+    public:
+        //dist_landing + dist_obstacle + 1/2 robot_length
+        constexpr static double SAFE_MINING_DISTANCE { 1.5 + 2.98 + 1.3/2 };
+        //mining_zone_width - digging_radius
+        constexpr static double MINING_LINE_LENGTH { 3.78 - 1.75 };
+        //length of landing zone
+        constexpr static double FINISH_LINE { 1.5 };
+
```

Comment:
I would like to see these values set up as inputs to the node so that we don't have to recompile later if we decide we want to tune them.

Reply:
personally I think it's overkill but sure.

Reply:
Done! Retained constness as well, but don't get any of the benefits of constexpr and compile time checking anymore.